<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editAboutWindow</class>
 <widget class="QDialog" name="editAboutWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit the Opening Schedule</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="eventitle">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>240</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="eventStatringTime">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>400</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="eventClosingTime">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>450</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="eventDescription">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>310</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="events">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>501</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addEvent">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>490</y>
     <width>471</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Add a New Event</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteEvent">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>540</y>
     <width>471</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete an Event</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select The Days The Library is Open in:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter The Event's Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>280</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter The Event's Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>380</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter The Event's Starting Date &amp; Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>430</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter The Event's Closing Date &amp; Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>580</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>You Only Need To Enter The Event's Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>5</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Events</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>581</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="sundayGroup">
        <property name="title">
         <string>Sunday</string>
        </property>
        <widget class="QTimeEdit" name="closingSun">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>50</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QTimeEdit" name="openingSun">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Opening Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Closing Time</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mondayGroup">
        <property name="title">
         <string>Monday</string>
        </property>
        <widget class="QTimeEdit" name="closingMon">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>50</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QTimeEdit" name="openingMon">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Opening Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Closing Time</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="tuesdayGroup">
        <property name="title">
         <string>Tuesday</string>
        </property>
        <widget class="QTimeEdit" name="closingTues">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>50</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QTimeEdit" name="openingTues">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Opening Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Closing Time</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="wednesdayGroup">
        <property name="title">
         <string>Wednesday</string>
        </property>
        <widget class="QTimeEdit" name="closingWednes">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>50</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QTimeEdit" name="openingWednes">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Opening Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Closing Time</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="thursdayGroup">
        <property name="title">
         <string>Thursday</string>
        </property>
        <widget class="QTimeEdit" name="closingThurs">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>80</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QTimeEdit" name="openingThurs">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Opening Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Closing Time</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="fridayGroup">
        <property name="title">
         <string>Friday</string>
        </property>
        <widget class="QTimeEdit" name="closingFri">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>70</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QTimeEdit" name="openingFri">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Opening Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Closing Time</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="saturdayGroup">
        <property name="title">
         <string>Saturday</string>
        </property>
        <widget class="QTimeEdit" name="closingSatur">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>60</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QTimeEdit" name="openingSatur">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>130</width>
           <height>40</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Opening Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Closing Time</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>99</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QCheckBox" name="sunday">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sunday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="monday">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Monday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="tuesday">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tuesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="wednesday">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wednesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="thursday">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Thursday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="friday">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Friday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="saturday">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Saturday</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
